lnclient/second_ark/second_ark.go

```diff
@@ -0,0 +1,373 @@
+package second_ark
```

Comment:
Minor: should we call it "bark" everywhere rather than "Bark" in some places and "SecondArk" elsewhere?